main.tex

```diff
@@ -15,35 +15,33 @@
 \maketitle
 
 \begin{abstract}
+  Bitcoin's design is based on pseudonymous identities, and a publicly verifiable ledger of transactions transferring value between them. Although this design lacks strong guarantees for privacy, it does not rule it out.
+  Despite the potential for private use, research has shown that users' activity can often be traced in practice.
+  This lack of inherent privacy has given rise to business models built on dragnet surveillance of Bitcoin users which is problematic for Bitcoin's fungibility.
 
-% TODO citations should not be included in the abstract, the abstract should be cleaned up, and the remaining information and citations split off into an introduction section with citations
+  A number of methods have been proposed to mitigate these issues. Among these, CoinJoin is an approach to structuring transactions which adds ambiguity and breaks some common assumptions that underlie heuristics used for surveillance.
+  Current implementations of CoinJoin transactions present a number of limitations which may partly explain their lack of widespread adoption.
```

Comment:
```suggestion
  Current implementations of CoinJoin transactions present a number of limitations which partly explains their lack of widespread adoption.
```

Reply:
stylistically this is out of place if we're going for an academic tone, if we want to claim that we must provide evidence, which seems out of scope, but with "may" there is no such burden

Reply:
Isn't this an obviously true premise? If something has serious limitations on how it can be used, then it will be used less than without those limitations.

Reply:
it's just a stylistic convention that says the same thing, but qualifies that we are not going to try and argue for it empirically because that's out of scope. for example i'd expect that sentence without "may" in a usability study of bitcoin privacy software that shows a clear link with evidence. here it seems out of place for me, because it kind of blurs the boundaries between the definite claims we're making, assumptions we *need* to make to even state these claims (e.g. how bitcoin works), and non-critical assumptions, which this one is. this is a perfectly valid assumption to make, but it's not an assumption we rely on so adding "may" makes it clear that the validity of the claims we do make doesn't *depend* on it being true